composer.json

```diff
@@ -27,35 +27,38 @@
         "doctrine/inflector": "^1.4 || ^2.0",
         "knplabs/knp-menu": "^3.1",
         "knplabs/knp-menu-bundle": "^3.0",
-        "sonata-project/block-bundle": "^4.0",
-        "sonata-project/doctrine-extensions": "^1.4",
+        "sonata-project/block-bundle": "^4.2",
+        "sonata-project/doctrine-extensions": "^1.6",
         "sonata-project/exporter": "^2.1",
-        "sonata-project/form-extensions": "^0.1.1 || ^1.4",
-        "sonata-project/twig-extensions": "^0.1.1 || ^1.3",
-        "symfony/asset": "^4.4",
-        "symfony/config": "^4.4",
-        "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4.3",
-        "symfony/doctrine-bridge": "^4.4",
-        "symfony/event-dispatcher": "^4.4",
+        "sonata-project/form-extensions": "^1.4",
+        "sonata-project/twig-extensions": "^1.3",
+        "symfony/asset": "^4.4 || ^5.0",
```

Comment:
You can bump this to 5.1, the 5.0 version is unsupported

Reply:
Changes have been made